### ISSUE TITLE
Grid no longer supports the `-d` flag

### ISSUE THREAD

**Issue by craigmdennis** [morrislaptop/capistrano-gridserver#5]
Deploying now fails with the following error:

`rm -d /home/152547/users/.home/domains/example.com/current stdout: Nothing written`

I emailed support and was told:

>Thank you for your inquiry. As described previously in support request 3853181, we're updating your Grid hosting. In this example, it seems that the rm command has changed, and that the -d flag is no longer supported. You can check this by running the following command via SSH : 

>man rm 

>You may need to alter your command to have the same functionality without the -d flag. Thank you for your understanding here. 

So the script needs to be updated

**Comment by sharpdot** [comment]
just hit this problem myself. fixed it by changing to unlink instead of rm -d. i attached a patch file with my fix for this and issue #4 (i had to change the file extension)
[gridserver.txt](https://github.com/morrislaptop/capistrano-gridserver/files/307922/gridserver.txt)

**Comment by giacomoalonzi** [comment]
@sharpdot I tried your solution, but at the end of deploy nothing happened. We need to specify something? Media Temple need html folder to show the site. We need to symlink the current folder of the project to example.com/html/ right?

Thank you.

**Comment by sharpdot** [comment]
@giacomoalonzi yes you need to symlink html to current as part of the setup. this gem doesn't do that but it's a 1 time thing. make sure you've added the task in your deploy.rb file like the main readme says. this is what i have

namespace :deploy do
  after :restart, :clear_cache do
    on roles(:web), in: :groups, limit: 3, wait: 10 do
    end
  end
  # run the gridserver task
  after 'deploy:publishing', 'gridserver:create_relative_symlinks'
end

**Comment by giacomoalonzi** [comment]
@sharpdot thank you for the reply.
Can you help me with symlink? Have you a guide or an chunk of code for this task? I'm not sysadmin or ruby expert. Thank You!

**Comment by sharpdot** [comment]
on media temple to symlink the html folder to current you'll do something like this
`cd domains/yourdomainname
mv html html-static
ln -s current html`

Only do this after your current link exists.
mv moves your html folder to html-static - and you can always move that back if needed.

**Comment by giacomoalonzi** [comment]
@sharpdot thank you, but it's possibile to automate this command with an ruby task?
